server/utils/azureAppInsights.ts

```diff
@@ -36,7 +36,7 @@ export function buildAppInsightsClient(name = defaultName()): TelemetryClient |
 
     defaultClient.addTelemetryProcessor(({ tags, data }, contextObjects) => {
       const operationNameOverride = contextObjects?.correlationContext?.customProperties?.getProperty('operationName')
-      if (operationNameOverride && data.baseData) {
+      if (operationNameOverride && data?.baseData && tags) {
```

Comment:
think this project has more strict null checking, was moaning that `data` and `tags` could be undefined